demos/notebooks/demo_pipeline_cnmfE.ipynb

```diff
@@ -264,22 +372,69 @@
     "                                'min_corr': min_corr,\n",
     "                                'min_pnr': min_pnr,\n",
     "                                'normalize_init': False,               # just leave as is\n",
-    "                                'center_psf': True,                    # leave as is for 1 photon\n",
+    "                                'center_psf': True,                    # True for 1p\n",
     "                                'ssub_B': ssub_B,\n",
     "                                'ring_size_factor': ring_size_factor,\n",
     "                                'del_duplicates': True,                # whether to remove duplicates from initialization\n",
     "                                'border_pix': bord_px})                # number of pixels to not consider in the borders)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cnmfe_model = cnmf.CNMF(n_processes=n_processes, \n",
+    "                        dview=cluster, \n",
+    "                        params=parameters)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "    <h2 >CNMF-E: The Ring Model</h2>\n",
+    "   Background activity is very ill-behaved with 1p recordings: it often fluctuates locally and is much larger in magnitude than the neural signals we want to extract. In other words, the large-scale background model used for CNMF is not sufficient for most 1p data. Hence, Pengcheng Zhou and others came up with a localized model of background activity for CNMFE: CNMFE represents the background at each pixel as the weighted sum of activity from a circle (or ring) of pixels a certain distance from that pixel. The distance of this ring from the reference pixel is set by the <em>ring_size_factor</em> parameter. This more complex pixel-wise background model explains why CNMFE is computationally more expensive than CNMF, and also why it works better to mop up large-scale localized background noise to find the neurons in your 1p data. \n",
+    "    \n",
+    "When you set <em>gnb</em> in the CNMF model (usually to 1 or 2), you are setting the number of global background components to use. The fact that you can get away with so few is testament to how well-behaved the background activity is in 2p recordings compared to 1p. When we set <em>gnb</em> to 0 in Caiman, this is a flag telling Caiman's back end to switch to the ring model of the background activity. \n",
```

Comment:
In the CNMF model, setting gnb (usually to 1 or 2) sets the number of ...

The likely need for so few demonstrates how 2p recordings generally have better-behaved background activity, 
compared to 1p.